### PR TITLE
File Added : Assignment 7 and 8

### PR DESCRIPTION
**Pull request by Oshin-250197**

### CODE REVIEW

**Review by roshnidurugwar — APPROVED** [review]

**Review by arshad05786 — APPROVED** [review]

File: Oshin1/JavaScript/Assignment7/Triangle.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                color: white;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="logincentrecard">
+            <div class="card">
+                <div class="container">
+                    <h1 style="text-align: center;">Triangle Pattern</h1>
+
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment7/Triangle.html]: remove extra line

File: Oshin1/JavaScript/Assignment8/Quick_sort.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                        quickSort(arr, low, pi - 1);
+                        quickSort(arr, pi + 1, high);
+                    }
+                }
+            </script>
+        </div>
+        </div>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment8/Quick_sort.html]: add new line at the end of file

File: Oshin1/JavaScript/Assignment8/Quick_sort.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                    if (low < high) {
+                        pi = partition(arr, low, high);
+                        quickSort(arr, low, pi - 1);
+                        quickSort(arr, pi + 1, high);
+                    }
+                }
+            </script>
+        </div>
+        </div>
+        </div>
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment8/Quick_sort.html]: closing these div at there potion not after script tag

File: Oshin1/JavaScript/Assignment8/Sort1.html [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+                                    let temp = array[i];
+                                    array[i] = array[j];
+                                    array[j] = temp;
+                                }
+                            }
+                        }
+                    };
+                </script>
+            </div>
+        </div>
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment8/Sort1.html]: close these div at there position not after script tag

File: Oshin1/JavaScript/Assignment8/Sort2.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+                    } else {
+                        for (let i = 0; i < arr.length; i++) {
+                            arr[i] = parseInt(arr[i] + arr[i] * 0.2);
+                            result.innerHTML += `<div>Total marks ${i + 1} : ${arr[i]}</div>`;
+                        }
+                    }
+                }
+            </script>
+            </div>
+        </div>
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment8/Sort2.html]: close these div at there position not after script tag

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**